snake

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```

Comment:
When do you use this?

Reply:
I use it constantly during development.

I'll start with `snake check` to see if any of the requirements need to be updated.

Then during development, I'll make code changes and `snake rebuild`, which since Python doesn't actually compile is just running `snake test` and `snake lint`. Rinse, repeat.

At the end of development, when filling out the tests for coverage, I'll modify the code and `snake coverage`. Rinse, repeat.

We've got a GitHub Action to build a Docker container and push it up to DockerHub now, but prior to that I'd run `snake docker` and push the container to our repo, for example when upgrading LTA in k8s.

Reply:
These would make very useful additions to the CircleCI config!

`do_check_task()` is more complex, is there a PiPY package out there that does this?